student-info-system/test_transcript.py

```diff
@@ -0,0 +1,28 @@
+from django.test import TestCase
+
+from sis.models import Major
+from sis.tests.utils import createAdmin, createStudent
+
+
+class AdminTranscriptViewTest(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super(AdminTranscriptViewTest, cls).setUpTestData()
```

Comment:
Note that major now has a point of contact. Failing to include this made some tests fail. Just a heads up.